An Untitled Mining Game/An Untitled Mining Game/objects/obj_inventory/Draw_64.gml

```diff
@@ -0,0 +1,92 @@
+if(!show_inventory) exit;
+
+//-------Inventory Back
+draw_sprite_part_ext(
+	spr_inv_UI, 0, cell_size, 0, inv_UI_width, inv_UI_height, 
+	inv_UI_x, inv_UI_y, scale, scale, c_white, 1
+);
+
+var info_grid = ds_player_info;
+
+draw_set_font(fnt_text_24);
+var c = c_black; 
+draw_text_color(info_x, info_y, info_grid[# 0, 0] + ": " + info_grid[# 0, 1], c,c,c,c, 1);
+
+draw_set_font(fnt_smalldigits);
+draw_text_color(
+	info_x + (192*scale) + ((15+18)*scale), info_y, 
+	string(info_grid[# 1, 1]), c,c,c,c, 1
+	);
+
+
+//------Inventory
+var ii, ix, iy, xx, yy, sx, sy, iitem, inv_grid;
```

Comment:
You've heard this from me before but I'd like to reiterate; please name your variable names clearly! "ii", "ix", "iy", and so on have an indication of what they're actually used for or doing.